package.json

```diff
@@ -106,5 +106,16 @@
 		"type-fest": "^4.20.1",
 		"type-flag": "^3.0.0",
 		"typescript": "^5.5.2"
+	},
+	"pnpm": {
+		"packageExtensions": {
+			"node-pty": {
+				"//": "https://github.com/microsoft/node-pty/issues/777",
+				"dependencies": {
+					"node-gyp": "11.0.0"
```

Comment:
Seems this issue is fixed in v11.2.0?

https://github.com/nodejs/node-gyp/commit/3e1cdd4630f6b32feb8ef704a7b1e8cbf8d8ff2f

Reply:
I'm very unsure, I'm on Linux and while the bug reported seems to be Windows specific, I still experienced issues out of the box that were gone with 11.0.0. 🤔

Reply:
Ooh interesting... And the issue persisted with 11.2.0?